### DISCUSSION TITLE
What is the purpose of alternatives param when streaming transcript?

### DISCUSSION THREAD

**Discussion by jtoh89** [deepgram/community#343] [General help] [answered]
referencing this doc: https://developers.deepgram.com/reference/streaming#responses

what happens when we request more alternatives > 1?

**Comment by lukeocodes** [comment, marked as answer]
Hi @jtoh89, alternatives are a legacy feature that enables us to run the transcription multiple times. This has the potential for our customers (or their end users) to pick the most accurate transcription for their usecase.

**Reply by jtoh89** [reply]
ok. will there be something replacing it? also, where can we keep up-to-date with new and legacy/sunsetting features?

**Reply by lukeocodes** [reply]
We will never deprecate a feature in such a way as to introduce breaking changes to our users. You're safe, and if we ever intend to do that there will be a long notice period (6 months, at least) and plent of communication before hand.
